code/modules/uplink/uplink_items.dm

```diff
@@ -1649,6 +1683,62 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 20
 	restricted_roles = list("Chaplain")
 	surplus = 5 //Very low chance to get it in a surplus crate even without being the chaplain
+	
+/datum/uplink_item/role_restricted/spanish_flu
+	name = "Spanish Flu Culture"
+	desc = "A bottle of cursed blood, full of angry spirits which will burn all the heretics with the fires of hell.\
+			At least. thats what the label says"
+	item = /obj/item/reagent_containers/glass/bottle/fluspanish
+	cost = 11
+	restricted_roles = list("Chaplain, Virologist")
+	
+/datum/uplink_item/role_restricted/beesease
+	name = "Beesease Culture"
+	desc = "A bottle of microbees, which will quickly breed in the stomach of hosts.\
+			At least, that's what the label says."
+	item = /obj/item/reagent_containers/glass/bottle/beesease
+	cost = 10
+	restricted_roles = list("Virologist")
+
+/datum/uplink_item/role_restricted/retrovirus
+	name = "Retrovirus Culture Bottle"
+	desc = "A bottle of contagious DNA bugs, which will manually rearrange the DNA of hosts.\
+			At least, that's what the label says."
+	item = /obj/item/reagent_containers/glass/bottle/retrovirus
+	cost = 14
+	restricted_roles = list("Virologist, Geneticist")
+
+/datum/uplink_item/role_restricted/pierrot_throat
+	name = "Pierrot's Throat Culture Bottle"
+	desc = "A bottle of illegal microscopic throat clowns, which will, with their combined voices, honk, and breed, within the host of the throat.\
+			At least, that's what the label says."
+	item = /obj/item/reagent_containers/glass/bottle/pierrot_throat
+	cost = 5
+	restricted_roles = list("Virologist, Clown")
+	
+/datum/uplink_item/role_restricted/random_disease
+	name = "Experimental Disease"
+	desc = "A random disease. Maybe you'll get lucky."
+	item = /obj/item/reagent_containers/glass/bottle/random_virus
+	cost = 1
```

Comment:
Couldn't they just buy a bunch of these and likely get a really good one

Reply:
No. Higher symptoms are rare to spawn and optimal diseases are rarer